proto/regen/ecocredit/v1/state.proto

```diff
@@ -359,4 +359,20 @@ message ClassFees {
 
   // fees is the fee required to create a credit class.
   repeated cosmos.base.v1beta1.Coin fees = 1;
+}
+
+// AllowedBridgeChains is a list of chains that are allowed to be used in
+// bridging operations. NOTE: chain_names MUST be converted to uppercase before
+// writing to and reading from this table in order to keep entries consistent.
+//
+// Since Revision 1
+message AllowedBridgeChains {
```

Comment:
We have historically kept state messages singular. I think we should stick with the current pattern and update here.

```suggestion
message AllowedBridgeChain {
```